staging/src/k8s.io/legacy-cloud-providers/gce/gce_instances.go

```diff
@@ -210,36 +210,6 @@ func (g *Cloud) InstanceShutdownByProviderID(ctx context.Context, providerID str
 	return false, cloudprovider.NotImplemented
 }
 
-// InstanceMetadataByProviderID returns metadata of the specified instance.
-func (g *Cloud) InstanceMetadataByProviderID(ctx context.Context, providerID string) (*cloudprovider.InstanceMetadata, error) {
```

Comment:
Safe to remove since instance v2 is disabled: 

https://github.com/kubernetes/kubernetes/blob/d3c1e81d5e6796db9389cd73073f48d6a37c4621/staging/src/k8s.io/legacy-cloud-providers/gce/gce.go#L662-L664